<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Object3DConfigView</class>
 <widget class="QWidget" name="Object3DConfigView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="MainLayout">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="LinesL">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Lines config:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LineTypeLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="LineTypeL">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="LineTypeSolidB">
           <property name="text">
            <string>Solid</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">LineTypeG</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="LineTypeDashedB">
           <property name="text">
            <string>Dashed</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">LineTypeG</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LineColorLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="LineColorL">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TripletWidget" name="LineColorTriplet" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="LineWidthLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="WidthL">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="LineWidthSB">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="PointsL">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Points config:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="PointTypeLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="PointTypeL">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="PointTypeNoneB">
           <property name="text">
            <string>None</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">PointTypeG</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="PointTypeCircleB">
           <property name="text">
            <string>Circle</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">PointTypeG</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="PointTypeSquareB">
           <property name="text">
            <string>Square</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">PointTypeG</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="PointWidthLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="SizeL">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="PointSizeSB">
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>10.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="PointColorLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="PointColorL">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TripletWidget" name="PointColorTriplet" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TripletWidget</class>
   <extends>QWidget</extends>
   <header location="global">TripletWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="PointTypeG"/>
  <buttongroup name="LineTypeG"/>
 </buttongroups>
</ui>
